src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -472,6 +472,7 @@
 
         :else
         {:status 200
+         :headers {"Content-Type" "application/octet-stream"}
```

Comment:
I'd be into doing that, though I interpreted the ticket as pretty directed toward just this response. If @camlow325 is cool with that being the scope of this work, I'll go ahead and update this accordingly.